### PR TITLE
Add issue templates

### LINKED ISSUE #177: Add issue templates for knoxite and website projects
We've discussed that issue templates could be useful in future.

### PR DESCRIPTION
**Pull request by penguwin**
Closes #177

### CODE REVIEW

**Review by craftamap — APPROVED** [review]
Lgtm!

**Comment by penguwin** [comment]
Pushed a typo fix

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: "🚀 Feature request"
+about: "If you have a feature request 💡"
```
Comment (craftamap) [inline .github/ISSUE_TEMPLATE/feature_request.md:3]: Maybe something along the lines "Tell us what you are missing" is better - having feature request in name as well as in about seems redundant. :)

But I'm also okay with the current solution - just an idea :)

**Review by craftamap — DISMISSED** [review]
I left one comment, but we also can merge it like this :)

**Review by craftamap — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (muesli) [reply .github/ISSUE_TEMPLATE/feature_request.md]: @craftamap Good idea!

**Pull request merged**